controllers/managedocs_controller.go

```diff
@@ -405,6 +411,8 @@ func (r *ManagedOCSReconciler) reconcilePhases() (reconcile.Result, error) {
 	// Update the status of the components
 	r.updateComponentStatus()
 
+	r.DeploymentType = strings.ToLower(r.DeploymentType)
```

Comment:
Maybe in the future, we will need to take deployment type into consideration when we `updateComponentStatus` or `checkUninstallCondition`  so this need to be the first thing in the function or even better move it outside of `reconcilePhases`